src/components/Addbook.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+export default class Addbook extends Component {
+  render() {
+    return (
+      <div>
+        <input type="text" />
+        <input type="text" />
+        <button type="submit">Submit</button>
+      </div>
```

Comment:
- [ ] It would be nice if you could have either a label for each field or a placeholder that tells what a field represents. As it is, one cannot tell which is which for the book title or author. Please work on that 👍